ansible_runner/utils/capacity.py

```diff
@@ -21,17 +26,24 @@ def get_mem_in_bytes():
         return error
 
 
-def get_uuid():
```

Comment:
Rather than deleting a public function, why not fall back to `uuid.getnode()` or `uuid.uuid4()`?

Reply:
Something like this may work:
```python
def get_uuid(path=''):
files = [
    Path('/var/lib/dbus/machine-id'),
    Path('/etc/machine-id'),
]

if path:
    files.append(Path(path))

for file in files:
    try:
        uuid = file.read_text()
        return uuid.strip()
     except FileNotFoundError:
        continue
        
else:
    return uuid.getnode()
```